build/README.md

```diff
@@ -542,6 +549,27 @@ Pulls down authentication information for kubectl against a cluster, name can be
 Creates a short lived access to Google Cloud container repositories, so that you are able to call
 `docker push` directly. Useful when used in combination with `make push` command.
 
+### Terraform
+
+Utilities for deploying a Kubernetes Engine cluster on Google Cloud Platform using `google` Terraform provider.
+
+#### `make terraform-init`
+Install `google` and `google-beta` terraform providers and authorize.
+
+#### `make gcloud-terraform-cluster`
+Run next command with your project ID specified:
+```
+GCP_PROJECT=<YOUR_PROJECT_ID> [GKE_PASSWORD="<YOUR_PASSWORD>"] make gcloud-terraform-cluster
```

Comment:
There is a default project in `gcloud` - I take it that these two things are not linked?

Reply:
They are linked, but I think here is better to have an option to explicitly choose, which project you are going to use.

Reply:
Finally removed the GCP_PROJECT as prerequisites , I am setting it using `gcloud` command if not set.